solidity/contracts/BancorNetwork.sol

```diff
@@ -594,11 +517,7 @@ contract BancorNetwork is TokenHolder, ContractRegistryClient, ReentrancyGuard {
 
             // first check if the converter in this step is newer as older converters don't even support the beneficiary argument
             if (stepData.isV28OrHigherConverter) {
-                // if affiliate fee is processed in this step, beneficiary is the network contract
-                if (stepData.processAffiliateFee)
-                    stepData.beneficiary = payable(address(this));
-                    // if it's the last step, beneficiary is the final beneficiary
-                else if (i == data.length - 1)
+                if (i == data.length - 1)
```

Comment:
Could you add brackets here?

Reply:
Done all across the file; Can you please re-review?